src/app/products/share-products/dividends-share-product/dividends.component.html

```diff
@@ -0,0 +1,54 @@
+<div class="container m-b-20" fxLayout="row" fxLayoutAlign="end" fxLayoutGap="20px">
+    <button mat-raised-button color="primary">
+      <fa-icon icon="plus"></fa-icon>&nbsp;&nbsp;
+      Create Dividend
+    </button>
+</div>
+<div class="container">
+
+  <div fxLayout="row">
+    <mat-form-field fxFlex>
+      <mat-label>Filter</mat-label>
+      <input matInput (keyup)="applyFilter($event.target.value)">
+    </mat-form-field>
+  </div>
+
+  <div class="mat-elevation-z8">
+
+    <table mat-table [dataSource]="dataSource" matSort>
+
+      <ng-container matColumnDef="name">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Name </th>
+        <td mat-cell *matCellDef="let dividends">{{ dividends.productData.name }}</td>
+      </ng-container>
+
+      <ng-container matColumnDef="dividendPeriodStartDate">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Dividend Period Start Date </th>
+        <td mat-cell *matCellDef="let dividends">{{ dividends.dividendPeriodStartDate | date }}</td>
```

Comment:
```suggestion
        <td mat-cell *matCellDef="let dividend">{{ dividend.dividendPeriodStartDate | date }}</td>
```
please make this change for all mat cell defs